spec/forms/webauthn_setup_form_spec.rb

```diff
@@ -276,37 +275,26 @@
       end
 
       context 'with two existing platform authenticators one with the same name' do
-        let(:user) do
-          user = create(:user)
-          user.webauthn_configurations << create(
-            :webauthn_configuration,
-            name: device_name,
-            platform_authenticator: true,
-            transports: ['internal', 'hybrid'],
-          )
-          user.webauthn_configurations << create(
-            :webauthn_configuration,
-            name: device_name,
-            platform_authenticator: true,
-            transports: ['internal', 'hybrid'],
-          )
-          user
-        end
-        let(:params) do
-          super().merge(
-            platform_authenticator: true,
-            transports: 'internal,hybrid',
+        let!(:user) do
+          create(
+            :user,
+            webauthn_configurations: create_list(
+              :webauthn_configuration,
+              2,
```

Comment:
If I'm understanding correctly - this creates the 2 configurations at the same time rather than how they were being created before? Using `create_list`.